ux.symfony.com/.twig-cs-fixer.dist.php

```diff
@@ -1,5 +1,44 @@
 <?php
 
-$config = new TwigCsFixer\Config\Config();
+use TwigCsFixer\Rules\Delimiter\BlockNameSpacingRule;
+use TwigCsFixer\Rules\File\DirectoryNameRule;
+use TwigCsFixer\Rules\File\FileExtensionRule;
+use TwigCsFixer\Rules\File\FileNameRule;
+use TwigCsFixer\Rules\Function\IncludeFunctionRule;
+use TwigCsFixer\Rules\Punctuation\TrailingCommaSingleLineRule;
+use TwigCsFixer\Rules\String\SingleQuoteRule;
+use TwigCsFixer\Rules\Variable\VariableNameRule;
+use TwigCsFixer\Rules\Whitespace\EmptyLinesRule;
+use TwigCsFixer\Rules\Whitespace\TrailingSpaceRule;
+use TwigCsFixer\Ruleset\Ruleset;
+use TwigCsFixer\Rules\Whitespace\BlankEOFRule;
+use TwigCsFixer\Standard\Twig;
+
+$uxRuleset = (new Ruleset())
+    ->addStandard(new Twig())
+    // Symfony standard (customized for Symfony UX)
+    ->addRule(new DirectoryNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components']))
+    ->addRule(new FileNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components'], optionalPrefix: '_'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileExtensionRule())
```

Comment:
You might prefer
```
->addStandard(new Symfony())
->overrideRule(new DirectoryNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components']))
->overrideRule(new FileNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components'], optionalPrefix: '_'))
->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
```

Reply:
I'm feeling directory and filename always go by two.... or is it a specific case right there ?

Reply:
> I'm feeling directory and filename always go by two

Yes